### ISSUE TITLE
kbfs not compatible with macOS Sequoia

### ISSUE THREAD

**Issue by rex4539** [keybase/client#27378]

**Comment by shallax** [comment]
Here we go again, new version that works with Sequoia in 3 months?

**Comment by bkonick** [comment]
I'm having this problem too. Submitted a log report with ID `6f04f1630e61123c2f02941c`

**Comment by aplsms** [comment]
have the same issue. my log id: 3fc972f0a5ac95f881d2d11c

**Comment by aplsms** [comment]
> It's almost like the Keybase people aren't looking at these issues

they are also automatically close such issues...

**Comment by aplsms** [comment]
so 15 is missing. Someone needs to build it......

```
$ ls /Library/Filesystems/kbfuse.fs/Contents/Extensions/
12 13 14
```

**Comment by bleachbezel** [comment]
macOS Sequoia 15.1 Beta (24B5046f)...it's not working with the OS either. :-(

**Comment by jkdelauney** [comment]
Tossing my log ID on the pile: e20c4497fa10cae2b753011c

**Comment by retributter** [comment]
There's no ETA I guess?

**Comment by marcofranssen** [comment]
No need to be this harsh at the maintainers, they have been fixing this for all prior releases.

See:

- https://github.com/keybase/client/issues/25951
- https://github.com/keybase/client/issues/24677
- https://github.com/keybase/client/issues/24636
- https://github.com/keybase/client/issues/25043

Hopefully soon someone can cross-compile this stuff for MacOS Sequoia.

**Comment by rodaddy** [comment]
I'll toss a ditto on this as well.  I did get it work'n for every OS up till now, so 🤞🏼

**Comment by joshblum** [comment]
addressed in https://github.com/keybase/client/pull/28028

**Comment by marcofranssen** [comment]
@joshblum kudos 🎁 for you! Thanks for the upgrade. Let me bump my OS 😄  and give it a test.

**Comment by shallax** [comment]
@marcofranssen before you do that, note it's not yet built/ released.

**Comment by marcofranssen** [comment]
> @marcofranssen before you do that, note it's not yet built/ released.

Woops, thanks for pointing that out. Will hold of for tiny bit then indeed 😅

**Comment by ThomasPedleyNHS** [comment]
It's now available as v6.5.0 on the official website or via Homebrew.

**Comment by shallax** [comment]
It's now available as v6.5.0 from the official website and via Homebrew.
